artifact/metadata.go

```diff
@@ -263,6 +263,31 @@ type ArtifactDepends struct {
 	ArtifactGroup     []string `json:"artifact_group,omitempty"`
 }
 
+type Error struct {
+	Err string
+}
+
+func (e *Error) Error() string {
+	return e.Err
+}
+
+var ErrCompatibleDevices *Error = &Error{"ArtifactDepends: Required field 'CompatibleDevices' not found"}
```

Comment:
Why not use standard error? `... = errors.New("ArtifactDepends...`

Reply:
Fair point. Initially I thought about adding more information to the error.

Removing.

Reply:
+1